architecture/api/README.md

```diff
@@ -1 +1,2 @@
-Tip; load the apispc.yaml file in https://editor.swagger.io for a realtime view of the API documentation while editing the yaml file.
+**Tip:** use [Swagger UI](https://petstore.swagger.io/?url=https://raw.githubusercontent.com/minvws/nl-covid19-notification-app-coordination/master/architecture/api/apispec.yaml) or [ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/minvws/nl-covid19-notification-app-coordination/master/architecture/api/apispec.yaml) to view the `apispec.yaml` file, [Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/minvws/nl-covid19-notification-app-coordination/master/architecture/api/apispec.yaml) can be used  for a realtime view of the API documentation while editing the yaml file.
```

Comment:
Maybe it's better if we put swagger-ui inside the repo and load it via github pages? I tried it but got a cors error on first attempt, but that was when the apispec file wasn't yet in the repo. (there's a howto here: https://peterevans.dev/posts/how-to-host-swagger-docs-with-github-pages/

Reply:
(meanwhile I'll merge this)